### ISSUE TITLE
JAWS treats menu items after a separator as a different menu group

### ISSUE THREAD

**Issue by patrickrodee** [FreedomScientific/VFO-standards-support#210]
### Summary

> JAWS is not including the menu items that appear after a separator in the total menu item count list.
> 
> Example:
> 1. Go to https://codepen.io/patty/pen/MMEqZB
> 2. Focus on the root menu

### Expected result

> I expect JAWS to announce "Passionfruit, one of nine."

### Actual result

> JAWS announces "Passionfruit, one of four"

### Versions
Windows 10 Enterprise 
FireFox Version: 67.0.4 (64-bit) 
JAWS Version 2018.1811.2 ILM

**Comment by stevefaulkner** [comment]
@patrickrodee what JAWS version and what browser/version?

**Comment by stevefaulkner** [comment]
updated test case https://codepen.io/stevef/pen/OeOVQQ?editors=1000

**Comment by patrickrodee** [comment]
Hey Steve, I updated the issue description with the version information. Thanks!

**Comment by stevefaulkner** [comment]
@patrickrodee wondering why you think this should be the behaviour?

> JAWS is not including the menu items that appear after a separator in the total menu item count list.

I tested with latest version of NVDA and Firefox. Got the same result as JAWS, use of the separator results in 2 lists of 4 items.

**Comment by patrickrodee** [comment]
My thinking was that since all items are part of one group, they should be counted as such. VoiceOver on Chrome announces that there are eight items, for example.

**Comment by stevefaulkner** [comment]
@patrickrodee both JAWS and NVDA report to sets of 4 menu items as that is what is exposed in the accessibility tree in Firefox, In chrome they report  1 set of 8 `menuitems`. So it is not an issue with JAWS it is an issue with the differing browser implementations. I am unsure which implementation is correct (`separator` splits 1 set into 2 or not).

I have filed an issue on the [core AAM spec](https://github.com/w3c/core-aam/issues/49)
